ext/cuda/limiters.jl

```diff
@@ -21,39 +21,26 @@ function compute_element_bounds!(
     ρ,
     ::ClimaComms.CUDADevice,
 )
-    S = size(Fields.field_values(ρ))
-    (Ni, Nj, _, Nv, Nh) = S
+    (_, _, Nv, _, Nh) = DataLayouts.universal_size(ρ)
```

Comment:
This appears to be a bug fix.

Reply:
Ah, no, all is fine. I copied incorrect code over, that turned out to never be used. That is now fixed in this PR also (in DataLayouts.)